tlatools/org.lamport.tlatools/test-model/ParameterizedInstantiation/ParamInstance.tla

```diff
@@ -0,0 +1,31 @@
+--------------------------- MODULE ParamInstance ---------------------------
+EXTENDS Naturals
+
+VARIABLE value
+
+  ------ MODULE Ring ------
+  VARIABLE cardinality
+  
+  Init == /\ cardinality > 1
+          /\ value \in (0 .. cardinality)
+          
+  Next == value' = value % cardinality
+             
+  Spec == /\ Init
+          /\ [][Next]_<<value, cardinality>>
+  
+  =================================
+  
+Z7(cardinality) == INSTANCE Ring
+
+SpecA == Z7(7)!Spec
```

Comment:
Did you also look into operator instantiation (see [section 5 of TLA+2 guide](https://lamport.azurewebsites.net/tla/tla2-guide.pdf)) or do you expect no special handling is required in TLC?

Reply:
I did not, will verify and address if kaput.

Reply:
[Section 2.1 of current-tools.pdf](http://lamport.azurewebsites.net/tla/current-tools.pdf) suggests a simple workaround.  To what extent does is this related?

Reply:
Not sure what you mean by related, but since the suggested workaround has a zero-length argument list, its functioning will not be affected by these code changes.

Reply:
This has not affected the handling of infix operator definitions; e.g
```
ModSanityCheck == Z7(7)!$$(5,9) < 7
```
where the inner module has this definition
```
  x $$ y == (x + y) % cardinality
```
evaluates correctly. I'm unable to create a correctly parsed prefix or postfix operator definition even in older builds of the Toolbox/TLATools, though TLA documentation implies this is supposed to work.